src/patternfly/components/Nav/nav.scss

```
@@ -1,1210 +1,415 @@
-  --#{$nav}__section__link--active--after--BorderWidth: 0;
-  --#{$nav}__section__link--m-current--after--BorderWidth: var(--#{$pf-global}--BorderWidth--xl);
-  --#{$nav}__section--section--MarginTop: var(--#{$pf-global}--spacer--xl);
-
-  // Nav section title
-  --#{$nav}__section-title--PaddingTop: var(--pf-t--global--spacer--sm);
-  --#{$nav}__section-title--PaddingRight: var(--pf-t--global--spacer--md);
-  --#{$nav}__section-title--PaddingBottom: var(--pf-t--global--spacer--sm);
-  --#{$nav}__section-title--PaddingLeft: var(--pf-t--global--spacer--md);
-  --#{$nav}__section-title--MarginBottom: var(--pf-t--global--spacer--sm);
-  --#{$nav}__section-title--xl--PaddingRight: var(--pf-t--global--spacer--xl);
-  --#{$nav}__section-title--xl--PaddingLeft: var(--pf-t--global--spacer--xl);
-  --#{$nav}__section-title--FontSize: var(--pf-t--global--font--size--body--default);
-  --#{$nav}__section-title--Color: var(--pf-t--global--text--color--regular);
-  --#{$nav}__section-title--BorderBottomColor: transparent;
-  --#{$nav}__section-title--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--sm);
-
-  // Scroll buttons
-  --#{$nav}__scroll-button--Color: var(--#{$pf-global}--Color--light-100);
-  --#{$nav}__scroll-button--hover--Color: var(--#{$pf-global}--active-color--400);
-  --#{$nav}__scroll-button--focus--Color: var(--#{$pf-global}--active-color--400);
-  --#{$nav}__scroll-button--active--Color: var(--#{$pf-global}--active-color--400);
-  --#{$nav}__scroll-button--disabled--Color: var(--#{$pf-global}--disabled-color--100);
-  --#{$nav}__scroll-button--BackgroundColor: transparent;
-  --#{$nav}__scroll-button--Width: var(--#{$pf-global}--target-size--MinWidth);
-  --#{$nav}__scroll-button--OutlineOffset: calc(-1 * var(--#{$pf-global}--spacer--xs));
-  --#{$nav}__scroll-button--Transition: margin .125s, transform .125s, opacity .125s;
-
-  // Scroll buttons before
-  --#{$nav}__scroll-button--before--BorderColor: var(--#{$pf-global}--BackgroundColor--dark-200);
-  --#{$nav}__scroll-button--before--BorderWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$nav}__scroll-button--before--BorderRightWidth: 0;
-  --#{$nav}__scroll-button--before--BorderLeftWidth: 0;
-  --#{$nav}__scroll-button--disabled--before--BorderColor: transparent;
-
-  // Toggle
-  --#{$nav}__toggle--PaddingRight: var(--#{$pf-global}--spacer--sm);
-  --#{$nav}__toggle--PaddingLeft: var(--#{$pf-global}--spacer--sm);
-  --#{$nav}__toggle--FontSize: var(--#{$pf-global}--icon--FontSize--md);
-
-  // Toggle icon
-  --#{$nav}__toggle-icon--Transition: var(--#{$pf-global}--TransitionDuration);
-
-  // Divider
-  --#{$nav}--c-divider--MarginTop: var(--#{$pf-global}--spacer--sm);
-  --#{$nav}--c-divider--MarginBottom: var(--#{$pf-global}--spacer--sm);
-  --#{$nav}--c-divider--PaddingRight: 0;
-  --#{$nav}--c-divider--PaddingLeft: 0;
-  --#{$nav}--c-divider--BackgroundColor: var(--#{$pf-global}--BackgroundColor--dark-200);
-
-  // Scroll snap
+:root,
+:where(.#{$nav}) {
```

Other components have been updated to root + just the component selector here:

```suggestion
:root,
.#{$nav} {
```

Do we want to shift to utilizing `:where` here?

---

The core team talked about this earlier this week. Using `:where()` here resolves some variables conflicts while allowing `vars()` to be updated/themed without wrapping in the `.pf-v5-c-component` selector.